### PR TITLE
fix dim font

### PR DESCRIPTION
**Pull request by glmnet**
Fixes thin font in iOS, I expect this to fix it in other OSs too.

### CODE REVIEW

File: src/editor/esphome-editor.ts [hunk 1]
```diff
@@ -61,6 +61,8 @@ export class ESPHomeEditor extends LitElement {
         enabled: false,
       },
       tabSize: 2,
+      fontFamily:
+        "SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace",
```
Comment (balloob) [inline src/editor/esphome-editor.ts:65]: Add Roboto Mono to it.

Comment (balloob) [reply src/editor/esphome-editor.ts:65]: Try this https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/#the-monospaced-system-ui-css-font-stack

Comment (balloob) [reply src/editor/esphome-editor.ts:65]: Let's just copy what Tailwind uses

https://github.com/tailwindlabs/tailwindcss/blob/master/src/css/preflight.css#L106

Comment (balloob) [reply src/editor/esphome-editor.ts:65]: Also don't wrap the whole value in quotes, that is not supported

Comment (glmnet) [reply src/editor/esphome-editor.ts]: too many quotes there

Comment (glmnet) [reply src/editor/esphome-editor.ts]: > Also don't wrap the whole value in quotes, that is not supported

this is not a css style rule, it's a js object

File: src/editor/esphome-editor.ts [hunk 2]
```diff
@@ -61,6 +61,8 @@ export class ESPHomeEditor extends LitElement {
         enabled: false,
       },
       tabSize: 2,
+      fontFamily:
+        "ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace",
```
Comment (balloob) [inline src/editor/esphome-editor.ts:65]: I think that liberation mono still needs to be wrapped in quotes to be a valid font family value, no ?
```suggestion
        "ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,\"Liberation Mono\",\"Courier New\",monospace",
```

**Review by balloob — APPROVED** [review]

File: src/editor/esphome-editor.ts (hunk 2 above)
Comment (glmnet) [reply src/editor/esphome-editor.ts]: you're right, spaces needs to be escaped

**Pull request merged**